src/shared/components/TextArea/TextArea.style.ts

```diff
@@ -1,67 +1,14 @@
-import { colors, sizes, transitions, typography } from '@/shared/theme'
 import styled from '@emotion/styled'
-
-export const TextAreaWrapper = styled.div`
-  width: 100%;
-`
+import resizer from '@/shared/icons/resizer.svg'
 
 export const StyledTextArea = styled.textarea`
-  background: transparent;
   width: 100%;
-  resize: none;
-  color: ${colors.gray[300]};
-  font-size: ${typography.sizes.body1};
-  line-height: ${typography.lineHeights.body1};
-  border: none;
-  border-bottom: 1px solid ${colors.gray[900]};
-  transition: all ${transitions.timings.loading} ${transitions.easing};
-  ::placeholder {
-    color: ${colors.gray[400]};
-    transition: color ${transitions.timings.loading} ${transitions.easing};
-  }
-  :focus {
-    border-bottom: 1px solid ${colors.blue[500]};
+  display: block;
+  position: relative;
+  :disabled {
+    cursor: not-allowed;
   }
-  :placeholder-shown {
-    border-bottom: 1px solid ${colors.gray[200]};
+  ::-webkit-resizer {
+    background: url(${resizer}) no-repeat;
```

Comment:
Wasn't aware you can do that, looks cool!

However, on Firefox it looks bad. It looked bad even before so it's not on your changes but I think we could improve that.

Our resize:
<img width="190" alt="Screenshot 2021-04-06 at 13 49 12" src="https://user-images.githubusercontent.com/12646744/113706702-4ab23b00-96df-11eb-9b74-a6e06e0bb745.png">

Default Firefox resize:
<img width="115" alt="Screenshot 2021-04-06 at 13 49 20" src="https://user-images.githubusercontent.com/12646744/113706731-5140b280-96df-11eb-9483-f05f22651ca1.png">

Could you look into why that happens and if there's something we could do to improve?

Reply:
Not sure if I follow here. On firefox, right now we are using a default resizer. You cannot do anything with it. It's impossible to change it and it's impossible to hide it. 
To do something with it I would have to disable resizing and implement resizing from the scratch, I think.
On other browsers, I've added a custom resizer using `::-webkit-resizer`,  but as you see, it's not ideal. I cannot change its position and size.

Reply:
Yes, we're using a default one, but for some reason it doesn't look as good as resize I've posted which is also a default one, just from a clean up. My guess is that some of our CSS may be causing it to look the way it does, be it margin, padding or something else. I'm asking if you can see what is the reason

Reply:
Not much we can do it seems, that's on Firefox